helpers.go

```diff
@@ -76,14 +76,15 @@ func ReadEnvFiles(envFiles []string, mergeWithEnv, ignoreMissing bool) (env []st
 
 // formatNixCode will format Nix code by calling 'nixfmt' and passing in the
 // given code via stdin.
-func formatNixCode(contents []byte) ([]byte, error) {
+func formatNixCode(contents []byte, formatter string) ([]byte, error) {
+	//
 	// Check for existence of 'nixfmt' in $PATH.
-	nixfmtPath, err := exec.LookPath("nixfmt")
+	formatterPath, err := exec.LookPath(formatter)
 	if err != nil {
-		return nil, fmt.Errorf("'nixfmt' not found in $PATH: %w", err)
+		return nil, fmt.Errorf("%w not found in $PATH: %w", formatter, err)
```

Comment:
A better error message:

```
fmt.Errorf("configured formatter %q not found in $PATH: %w", formatter, err)
```

`%q` will wrap the string in double quotes (`"`).

Also, `%w` is only used to wrap errors in Go.